contrib/lib.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash -eEx
```

Comment:
can we put these funcs in az-helpers.sh?

Reply:
Found `buildlib/tools/common.sh` which is independent from `az`, added functions there instead of new `lib.sh`.
Proposal is to not add new duplication with this PR, but also not do further clean ups in `test_jenkins.sh`. sourcing on top of script should avoid collateral. Later more incremental reworks of `test_jenkins.sh` could be performed.